packages/kvib-react-button/src/style.css

```diff
@@ -0,0 +1,140 @@
+@import url("https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200");
```

Comment:
Vi kan ikke ha egen CSS i react-pakkene, de må bruke CSSen fra kvib/css for at vi skal ha én kilde til sannhet. Hva skjer når man oppdaterer CSS'en i kvib/css, da vil man måtte manuelt oppdatere CSS'en her også liksom

Reply:
Joda jeg er helt enig, og det er tanken etterhvert selvsagt. Men for å ikke brekke eksisterende kvib cssen for knapper så har vi inntil videre plassert cssen for den nye knappen i egen fil i react-komponenten.

Reply:
Sebastian jobber med å oppdatere css-en for knappene i css-biblioteket, så dette kommer snart på plass :)